playbooks/ansible/roles/sit.cephfs/templates/ceph.smb.cluster.yml.j2

```diff
@@ -0,0 +1,39 @@
+resources:
+- resource_type: ceph.smb.usersgroups
+  users_groups_id: siteacct
+  intent: present
+  values:
+    {%- for account_group in config.accounts +%}
+      {%- if config.accounts[account_group].users +%}
+      users:
```

Comment:
It seems that users and groups are created but users are not assigned to group.s

Reply:
[ceph.smb.usersgroups](https://docs.ceph.com/en/latest/mgr/smb/#users-and-groups-resource) resource doesn't allow us to specify such groups to which a user can be assigned.

@phlogistonjohn any thoughts?

Reply:
I keep forgetting about that. The AD DC config has this but apparently the user/groups config of the file server is still missing it. Please file an issue on sambacc.

Reply:
In that case we can leave this PR as is, and add user-group assignment later when it's supported by the smb mgr module.

Reply:
> In that case we can leave this PR as is, and add user-group assignment later when it's supported by the smb mgr module.

Created #115 so that we don't forget once the support is in place.